src/internal/m365/service/onedrive/backup.go

```diff
@@ -123,6 +126,8 @@ func migrationCollections(
 		return nil, nil
 	}
 
+	counter.Inc("requires_migrate_user_pn_to_id")
```

Comment:
This is missing a key name

Reply:
heh, you can tell I was wearing out by this point.  The work is getting sloppy 😆